include/swift/Reflection/ReflectionContext.h

```diff
@@ -644,6 +645,47 @@ class ReflectionContext
     return false;
   }
 
+  /// Adds an image using the FindSection closure to find the swift metadata
+  /// sections. \param[in] FindSection
+  ///     Closure that finds sections by name. ReflectionContext is in charge
+  ///     of freeing the memory buffer in the RemoteRef return value.
+  ///     process.
+  /// \return
+  ///     /b True if any of the reflection sections were registered,
```

Comment:
I'm not super familiar with Doxygen conventions so maybe this is correct: is this supposed to be a forward-slash in `/b`?

Reply:
Tested it out and you're right, it should be `/b`